documentation/installation.md

```diff
@@ -30,6 +32,20 @@ LOG_LEVEL    | Numeric value indicating the log level.  10 is verbose, 20 is inf
 
 Once everything is set up, to run standup-bot, simply run `npm start`.
 
+## Docker
```

Comment:
Great!  Thanks for writing out extra documentation!  This is very helpful.  :smile: